source/us/index.blade.php

```diff
@@ -257,10 +257,13 @@
 -->
 <section id="participation" class="mx-auto py-5 col-lg-11">
 	<div class="row">
-		<div class="mont-700 h1 underline position-relative mx-5 pb-4">How to participate?</div>
+		<div class="mont-700 h1 position-relative mx-5 pb-4">How to participate?</div>
 	</div>
 	<div class="row py-4">
```

Comment:
this py-4 should be removed (only last div with class row should contain this class)